src/test/scala/org/kibanaLoadTest/scenario/TelemetryAPI.scala

```diff
@@ -0,0 +1,34 @@
+package org.kibanaLoadTest.scenario
+
+import cats.implicits.catsSyntaxSemigroup
+
+import io.gatling.core.Predef.{exec, _}
+import io.gatling.http.Predef._
+
+object TelemetryAPI {
+  def load(baseUrl: String, headers: Map[String, String]) = {
+    val defaultHeaders =
+      headers.combine(Map("Referer" -> s"$baseUrl/app/home"))
+
+    exec(
+      http("telemetry: /api/telemetry/v2/clusters/_stats")
+        .post("/api/telemetry/v2/clusters/_stats")
+        .body(StringBody("""{ "refreshCache": true }""")) // Request for fresh cache so we can measure the effect of generating the telemetry report multiple times
```

Comment:
How often does Kibana call the API with `refreshCache` flag? 
Do we emulate a scenario when multiple Kibana instances request ES due to missing cache?

Reply:
I created this _always-fresh_ request to monitor how badly the generation of a new report affects. Kind of indirectly measures how much load can we create on ES & Kibana at once during the report generation. This way we can correlate with the advantages of having the cache mechanism, but it also can give us a hint about the number of concurrent report generations Kibana can handle (it may be important to understand this number when troubleshooting other performance issues).

> Do we emulate a scenario when multiple Kibana instances request ES due to missing cache?

I don't think that we can do that in this suite, can we? If we could, that would be great because it'll help us identify the limits on ES.